src/loader/loader.js

```diff
@@ -688,7 +695,9 @@
          * @return {HTMLImageElement}
          */
         api.getImage = function (image) {
-            if ((image instanceof HTMLImageElement) || (image instanceof HTMLCanvasElement)) {
```

Comment:
yeah but if you remove this, this is breaking the `getImage()` behaviour for a lots of things around the framework.... why not rather add `if (weixinSmallGame && ! String)` ?